inginious/frontend/template_helper.py

```diff
@@ -43,6 +43,8 @@ def __init__(self, plugin_manager, user_manager, default_template_dir, default_l
                                                                                            **kwargs)),
                               "superadmin_options": (
                                   lambda **kwargs: self._generic_hook('superadmin_options', **kwargs)),
+                              "uncode_navbar_options": (
```

Comment:
Maybe is a good idea to create a page with a simple explanation about the new hooks